docs/conf.py

```diff
@@ -15,7 +15,46 @@
 
 # -- Project information -----------------------------------------------------
 
+import os
+import sys
+import tempfile
+import importlib
+import subprocess
+
+sys.path.append(os.path.abspath('..'))
+
 import honeycomb
+import services
+import integrations
+
+from honeycomb.servicemanager import base_service
+from honeycomb.integrationmanager import integration_utils
+
+services_list = next(os.walk('../services'))[1]
+integrations_list = next(os.walk('../integrations'))[1]
+
+
+def install_plugins(dir, plugins, target):
+    for plugin in plugins:
+        reqs = os.path.join(dir, plugin, "requirements.txt")
+        if os.path.exists(reqs):
+            pipargs = ["install", "-r", reqs, "--target", target, "--ignore-installed"]
+            subprocess.check_call([sys.executable, "-m", "pip"] + pipargs)
+
+
+venv = tempfile.mkdtemp()
+install_plugins('../services', services_list, venv)
+install_plugins('../integrations', integrations_list, venv)
+
+sys.path.append(venv)
+sys.path.append(os.path.dirname(honeycomb.__file__))
+sys.path.append(os.path.dirname(base_service.__file__))
+sys.path.append(os.path.dirname(integration_utils.__file__))
+
+for _ in services_list:
+    importlib.import_module('services.{}'.format(_))
```

Comment:
It's worth it to catch import errors here, just in case. And 2 lines under.

Reply:
if import doesn't work than sphinx won't be able to generate the document, so it's better to break and alert us something is wrong
again this is only used during readthedocs build